docs/guides/migrate-rest-myst.md

```diff
@@ -0,0 +1,144 @@
+# Migrating from reStructuredText to MyST Markdown
+
+Sphinx is usually associated with reStructuredText, the markup language
+{pep}`designed for the CPython project in the early '00s <287>`.
+However, for quite some time Sphinx has been compatible with Markdown as well,
+thanks to a number of extensions.
+
+The most powerful of such extensions is [MyST-Parser], which implements
+a CommonMark-compliant, extensible Markdown dialect with support for the Sphinx
+roles and directives that make it so useful. In this guide, you will find
+how you can start writing Markdown in your existing reStructuredText project,
+or migrate it completely.
+
+If, instead of migrating, you are starting a new project from scratch,
+have a look at {doc}`myst-parser:sphinx/intro`.
+
+[MyST-Parser]: https://myst-parser.readthedocs.io/
+
+## Writing your content both in reStructuredText and MyST
+
+It is useful to ask whether a migration is necessary in the first place.
+Doing bulk migrations of large projects with lots of work in progress
+will create conflicts for ongoing changes.
+On the other hand, your writers might prefer to have some files in Markdown
+and some others in reStructuredText, for whatever reason.
+Luckily, Sphinx supports reading both types of markup at the same time without problems.
+
+To start using MyST in your existing Sphinx project,
+first {ref}``install the `myst-parser` Python package <myst-parser:sphinx/intro.md#install-the-myst-parser>``
+and then {ref}`enable it on your configuration <myst-parser:parse-with-sphinx>`:
+
+```{code-block} py
+:caption: conf.py
+:emphasize-lines: 4
+
+extensions = [
+    # Your existing extensions
+    ...,
+    "myst_parser",
+]
+```
+
+Your reStructuredText documents will keep rendering,
+and you will be able to add MyST documents with the `.md` extension
+that will be processed by MyST-Parser.
+
+As an example, _this guide_ is written in MyST
+while the rest of the Read the Docs documentation is written in reStructuredText.
+
+```{note}
+By default, MyST-Parser registers the `.md` suffix for MyST source files.
+If you want to use a different suffix, you can do so by changing your
+`source_suffix` configuration value in `conf.py`.
+```
+
+## Converting existing reStructuredText documentation to MyST
+
+To convert existing reST documents to MyST, you can use
+the `rst2myst` CLI script shipped by [rst-to-myst].
+The script supports converting the documents one by one,
+or scanning a series of directories to convert them in bulk.
+
+After {doc}``installing `rst-to-myst` <rst-to-myst:index>``,
+you can run the script as follows:
+
+```console
+$ rst2myst convert docs/source/index.rst  # Converts index.rst to index.md
+$ rst2myst convert docs/**/*.rst  # Convert every .rst file under the docs directory
+```
+
+This will create a `.md` MyST file for every `.rst` source file converted.
+
+[rst-to-myst]: https://rst-to-myst.readthedocs.io/
```

Comment:
Is this not a reference to the index file?

Reply:
Right. I was not using intersphinx for the index page until now, but it's better for consistency. Changing.